src/main/scala/io/iohk/praos/domain/BlockchainState.scala

```diff
@@ -0,0 +1,7 @@
+package io.iohk.praos.domain
+
+import akka.util.ByteString
+
+case class BlockchainState(blockchain: Blockchain, maybeHeadBlockHash: Option[ByteString], receivedChains: List[Blockchain])
```

Comment:
As `maybeHeadBlockHash` is obtainable from blockchain, maybe it could be defined as a function or a `lazy val`?

Reply:
:+1: